mappings/markdown/Zeek/v1.3.0/rdp_log/README.md

```diff
@@ -0,0 +1,74 @@
+
+
+ ### Conditional mapping:
+Fields described here are subject to dynamic mappings contingent on a conditional evaluation of source data.
+| OCSF                           | Raw               | Zeek Field Description                                              | Evaluation Conditions                                                                               |
+| ------------------------------ | ----------------- | ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| `status_id`                    | `auth_success`    | Authentication result.                                              | If "true" then "1", <br>if "false" then "2", <br>else "0". <br>Type is Integer.                    |
+
+
+ ### Unmapped:
+| OCSF                     | Raw                | Zeek Field Description                                                                  |
+| -------------------------| -------------------| --------------------------------------------------------------------------------------- |
+| `unmapped`               | `cert_count`       | The number of certs seen.                                                               |
+| `unmapped`               | `cert_permanent`   | Indicates if the provided certificate or certificate chain is permanent or temporary.   |
+| `unmapped`               | `cert_type`        | If the connection is being encrypted with native RDP encryption, this is the type of cert being used. |
+| `unmapped`               | `channels_joined`  | The number of channels a client joined during the connection sequence.                  |
+| `unmapped`               | `client_channels`  | The channels requested by the client.                                                   |
+| `unmapped`               | `encryption_level` | Encryption level of the connection.                                                     |
+| `observables[].value`          | `rdfp_hash`       | The rdfp_hash information.                                                              | In an object where `observables[].type_id` = "30", <br>and `observables[].name` = "rdfp_hash".       |
+| `observables[].value`          | `rdfp_string`     | A fingerprint which represents an RDP client.                                           | In an object where `observables[].type_id` = "99", <br>and `observables[].type` = "RDP client fingerprint" <br>and `observables[].name` = "rdfp_string".          |
+| `observables[].value`          | `keyboard_layout` | Keyboard layout (language) of the client machine.                                       | In an object where `observables[].type_id` = "99", <br>and `observables[].type` = "Client machine keyboard layout language",  <br>and `observables[].name` = "keyboard_layout".          |
```

Comment:
We have a `keyboard_info` object which contains this attribute. Likely we should add this to the RDP event class